cmd/cli/namespace_add.go

```diff
@@ -142,3 +151,17 @@ func (a *namespaceAddCmd) run() error {
 
 	return nil
 }
+
+func getMeshes(clientSet kubernetes.Interface, meshName string, namespace string) ([]v1.Deployment, error) {
```

Comment:
Is it possible that we deployed multiple meshes in one namespace? What's our assumption here?

If getMeshes() returns multiple meshes, why the `meshName` is given here?

Reply:
There cannot be multiple meshes in the same control plane namespace. Multiple instances of the control plane can exist as replicas within a single mesh though.

Reply:
@allenlsy and @shashankram sorry the name wasn't clear. What this does is check if the mesh with `meshName` is present in the cluster by getting `meshName`'s controllers. If the controllers for `meshName` exists, then that means `meshName` exists. This means we can proceed to add `namespace` to be managed by the mesh `meshName`.

If there are no controllers, then that means the mesh `meshName` does not exist.

![image](https://user-images.githubusercontent.com/13926417/117059668-e6ca6480-acd4-11eb-8fae-86d4a8fadf80.png)
![image](https://user-images.githubusercontent.com/13926417/117059721-f3e75380-acd4-11eb-8dd8-77b9fdea0578.png)